eo-maven-plugin/src/test/java/org/eolang/maven/ResolveMojoTest.java

```diff
@@ -66,4 +70,86 @@ public void testSimpleResolve(@TempDir final Path temp) throws Exception {
         );
     }
 
+    @Test
+    public void testConflictingDependencies(@TempDir final Path temp) {
+        final Exception excpt = Assertions.assertThrows(
+            IllegalStateException.class,
+            () -> simulateConflicts(temp, false)
+        );
+        MatcherAssert.assertThat(
+            excpt.getMessage(),
+            Matchers.equalTo(
+                String.format(
```

Comment:
@mximp what's the point of this string composition here? Can't you just use a single literal?

Reply:
@yegor256 single literal is too long. Hence I split it into two lines.
Probably `String.join` is better alternative here.

Reply:
@mximp I suggest just suppressing Checkstyle for this particular line. it will look most appropriate